packages/url/init.lua

```diff
@@ -147,11 +150,12 @@ function package:registerCommands ()
         secondaryPenalty = secondaryPenalty,
         worsePenalty = worsePenalty
       })
-      SILE.call("code", {}, result)
+
+      SILE.call("urlstyle", {}, result)
     end)
   end, "Inserts penalties in an URL so it can be broken over multiple lines at appropriate places.")
 
-  self:registerCommand("code", function (options, content)
```

Comment:
So we need to move `code` to the plain class, or plenty of packages documentation will break, etc. It's non visible in the manual, because it redefines it in its macros, but several packages make use of it.

I wonder whether `urlstyle` should call that `code` or directly `verbatim:font` (which is from the **verbatim** package, which url loads... Hmm that makes a possible reason to call it directly here.. The implementation of code becomes trickier, though... Er... Maybe `verbatim:font` should _only_ be used in the context where verbatim is actually really used (One can imagine a different font being used for code in text flow vs. code in verbatim blocks).